### PR TITLE
Recursive Code To Get Java Code to Parse Json

### PR DESCRIPTION
**Pull request by vipul-08**

### CODE REVIEW

**Comment by omkarprabhu-continuum** [comment]
@vipul-08 
- Add your name in Contributors
- Change all the _nerdyninja_ specific URLs to _**prabhuomkar**_

I came across a bug (which is not a part of scope of this PR)

**Comment by prabhuomkar** [comment]
@vipul-08 
- Add your name in Contributors
- Change all the _nerdyninja_ specific URLs to _**prabhuomkar**_

I came across a bug (which is not in the scope of this PR), will create an issue. Please try to fix it as well :)

File: js/jsoncoder.js [hunk 1]
```diff
@@ -52,41 +41,59 @@ $(document).ready(function() {
...
+        else {
+          ans = ans + getType(objVal) + " " + objNow.replace(/[^A-Za-z]/g, "") + " = " + label + "." + getAction(objVal) + "(\"" + objNow + "\");" + "<br>";
+        }
       }
     }
-    // Send the code
-    $('#java-code').html(javaCode);
   }
 
+  // Thanks for Reading :-)
```
Comment (prabhuomkar) [inline js/jsoncoder.js]: Edit this line.
Replace it with some fancy quote :)

File: js/jsoncoder.js [hunk 2]
```diff
@@ -52,41 +41,59 @@ $(document).ready(function() {
...
+    }
+    $('#java-code').html(ans);
+  }
+
+  function parserJSON(jparsed, label) {
+    if (jparsed instanceof Array) {
+      ans = ans + "for(JSONObject insideObject:  " + label + ") {" + "<br>";
+      parserJSON(jparsed[0], "insideObject");
+      ans = ans +  "}<br>";
+
```
Comment (prabhuomkar) [inline js/jsoncoder.js]: Remove empty line

**Review by prabhuomkar — CHANGES_REQUESTED** [review]
Need to add `<br>` after several lines.

**Comment by vipul-08** [comment]
@prabhuomkar The changes are done. You can cross check and merge them.

**Review by prabhuomkar — APPROVED** [review]

**Pull request merged**